[main] Upgrade to latest dependencies

knative-automation opened the pull request:
Cron -knative-prow-robot

/cc knative-sandbox/client-writers
/assign $knative-sandbox/client-writers

Produced by: knative-sandbox/knobots/actions/update-deps

Details:
```
bumping knative.dev/client 33e2c9f...faee441:
  > faee441 Fix for serviceaccounts "default" not found flaky issue (# 1306)
  > fb07e25 Update knative/community files (# 1309)
  > adb6bdf Fully qualified test images (# 1307)
  > 7109533 Update actions (# 1303)
  > ccf721f upgrade to latest dependencies (# 1302)
  > 5ce3d1b chore: fixed documentation of configuration options in README (# 1297)
  > e5b19e1 Update Eventing Sources API to v1 (# 1299)
  > dccf3e3 upgrade to latest dependencies (# 1296)
  > e829ed2 upgrade to latest dependencies (# 1292)
  > 73f8219 Drop --strict flag on ko (# 1295)
  > eab8a57 upgrade to latest dependencies (# 1291)
  > 1c2381e upgrade to latest dependencies (# 1286)
  > 47a04cb Update knative/community files (# 1290)
  > 405691d Fix domain list output (# 1288)
  > cdc1d4e Add number of instances to describe command (# 1289)
  > 563f1d9 Fix service/revision list output with '-o' param (# 1276)
  > 64c6851 Update knative/community files (# 1279)
  > 33deb82 chore(ci): Update E2E latest release versions for Serving & Eventing (# 1285)
  > 910d136 Upgrade Eventing API from v1beta1 to v1 (# 1284)
bumping knative.dev/serving 0cc2c38...22f61ac:
  > 22f61ac Add an e2e hpa test (# 11188)
  > 330f4f5 Add test for periodSeconds>0 (# 11328)
  > 22ff014 Update net-kourier nightly (# 11334)
  > 4440449 Update net-istio nightly (# 11330)
  > 96c7bc1 Default webhook rule (# 11291)
  > 8162790 change type of revision status replicas (# 11310)
  > 1915d85 Fix markdown (# 11327)
  > 95245e5 Update knative/community files (# 11326)
  > 3a38d70 Add Domainmapping AutoTLS e2e test (# 10795)
```

maximilien reviewed (dismissed):
/ok-to-test
/approve
/lgtm

knative-automation committed:
upgrade to latest dependencies

bumping knative.dev/client 33e2c9f...faee441:
  > faee441 Fix for serviceaccounts "default" not found flaky issue (# 1306)
  > fb07e25 Update knative/community files (# 1309)
  > adb6bdf Fully qualified test images (# 1307)
  > 7109533 Update actions (# 1303)
  > ccf721f upgrade to latest dependencies (# 1302)
  > 5ce3d1b chore: fixed documentation of configuration options in README (# 1297)
  > e5b19e1 Update Eventing Sources API to v1 (# 1299)
  > dccf3e3 upgrade to latest dependencies (# 1296)
  > e829ed2 upgrade to latest dependencies (# 1292)
  > 73f8219 Drop --strict flag on ko (# 1295)
  > eab8a57 upgrade to latest dependencies (# 1291)
  > 1c2381e upgrade to latest dependencies (# 1286)
  > 47a04cb Update knative/community files (# 1290)
  > 405691d Fix domain list output (# 1288)
  > cdc1d4e Add number of instances to describe command (# 1289)
  > 563f1d9 Fix service/revision list output with '-o' param (# 1276)
  > 64c6851 Update knative/community files (# 1279)
  > 33deb82 chore(ci): Update E2E latest release versions for Serving & Eventing (# 1285)
  > 910d136 Upgrade Eventing API from v1beta1 to v1 (# 1284)
bumping knative.dev/serving 0cc2c38...22f61ac:
  > 22f61ac Add an e2e hpa test (# 11188)
  > 330f4f5 Add test for periodSeconds>0 (# 11328)
  > 22ff014 Update net-kourier nightly (# 11334)
  > 4440449 Update net-istio nightly (# 11330)
  > 96c7bc1 Default webhook rule (# 11291)
  > 8162790 change type of revision status replicas (# 11310)
  > 1915d85 Fix markdown (# 11327)
  > 95245e5 Update knative/community files (# 11326)
  > 3a38d70 Add Domainmapping AutoTLS e2e test (# 10795)

Signed-off-by: Knative Automation <automation@knative.team>

maximilien reviewed (approved):
/ok-to-test
/approve
/lgtm
